test/models/restaurant_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class RestaurantTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
If I said remove unnecessary files you have to remove all unnecessary files.
Why you always remove only those file which i commented on? 

I think It's 5th time i am explaining you !
![image](https://user-images.githubusercontent.com/97951500/187889976-1f21daf0-9632-44a5-98c6-dfb6d5d2dad9.png)